services/api/src/resources/task/helpers.ts

```diff
@@ -175,7 +177,9 @@ export const Helpers = (sqlClientPool: Pool) => ({
       environment: environmentData,
       advancedTask: {
         RunnerImage: image,
-        JSONPayload: new Buffer(JSON.stringify(payload).replace(/\\n/g, "\n")).toString('base64')
+        JSONPayload: new Buffer(JSON.stringify(payload).replace(/\\n/g, "\n")).toString('base64'),
+        deployerToken: adminTask, //an admintask will have a deployer token and ssh key injected into it
```

Comment:
The support for this isn't in any released controllers yet, it has only been merged into main pending a proper release.

Reply:
This is now supported in the controller